test/IntegrationTests/BuildTests.DistributionStructure_windows.verified.txt

```diff
@@ -246,10 +264,19 @@
   \store\x86\net6.0\microsoft.extensions.options.configurationextensions\6.0.0\lib\netstandard2.0\Microsoft.Extensions.Options.ConfigurationExtensions.dll,
   \store\x86\net6.0\microsoft.extensions.options\6.0.0\lib\netstandard2.1\Microsoft.Extensions.Options.dll,
   \store\x86\net6.0\microsoft.extensions.primitives\6.0.0\lib\net6.0\Microsoft.Extensions.Primitives.dll,
-  \store\x86\net6.0\mongodb.bson\2.3.0\lib\netstandard1.5\MongoDB.Bson.dll,
-  \store\x86\net6.0\mongodb.driver.core.extensions.diagnosticsources\1.1.0\lib\netstandard2.0\MongoDB.Driver.Core.Extensions.DiagnosticSources.dll,
-  \store\x86\net6.0\mongodb.driver.core\2.3.0\lib\netstandard1.5\MongoDB.Driver.Core.dll,
+  \store\x86\net6.0\mongodb.bson\2.13.3\lib\netstandard2.1\MongoDB.Bson.dll,
+  \store\x86\net6.0\mongodb.driver.core.extensions.diagnosticsources\1.2.0\lib\netstandard2.0\MongoDB.Driver.Core.Extensions.DiagnosticSources.dll,
+  \store\x86\net6.0\mongodb.driver.core\2.13.3\lib\netstandard2.1\MongoDB.Driver.Core.dll,
+  \store\x86\net6.0\mongodb.driver.core\2.13.3\runtimes\win\native\libzstd.dll,
+  \store\x86\net6.0\mongodb.driver.core\2.13.3\runtimes\win\native\snappy32.dll,
+  \store\x86\net6.0\mongodb.driver.core\2.13.3\runtimes\win\native\snappy64.dll,
+  \store\x86\net6.0\mongodb.libmongocrypt\1.2.2\lib\netstandard2.1\MongoDB.Libmongocrypt.dll,
+  \store\x86\net6.0\mongodb.libmongocrypt\1.2.2\runtimes\linux\native\libmongocrypt.so,
+  \store\x86\net6.0\mongodb.libmongocrypt\1.2.2\runtimes\osx\native\libmongocrypt.dylib,
+  \store\x86\net6.0\mongodb.libmongocrypt\1.2.2\runtimes\win\native\mongocrypt.dll,
+  \store\x86\net6.0\sharpcompress\0.23.0\lib\netstandard2.0\SharpCompress.dll,
   \store\x86\netcoreapp3.1\artifact.xml,
+  \store\x86\netcoreapp3.1\dnsclient\1.4.0\lib\netstandard2.1\DnsClient.dll,
```

Comment:
Might need to start documenting possible conflicting dependencies.
I see it brings 3 new dependencies >
* mongodb.libmongocrypt
* sharpcompress
* dnsclient

More worried about last 2.

Reply:
New issue created: #1380